### ISSUE TITLE
Not sure if bug or feature - eggs don't hatch when dispensed

### ISSUE THREAD

**Issue by Raavea** [Okabintaro/UntitledDuckMod#29]
**Describe the bug**
I'm not sure if this is a bug, or a feature of the mod, or a limitation of the way mods work, but;
 I have a world I'm playing where we've got a big flock of ducks from your mod - not purposefully, but there's LOADS of them, and I haven't seen any chickens, so we just decided to farm the ducks for meat.
 Earlier today I wanted to make an automated egg-hatching area, because doing it by hand takes a while, and eventually I might build a cooker, so these blocks can be integrated into that. However, I realised that when fired by a dispenser, your eggs just drop as items. At first, I thought this was being caused by my 'better dropped items' mod, but once I had the chance to test a world with that mod removed, I realised it's just the behaviour of the duck and goose eggs from this mod in particular.

TL;DR: Duck and Goose Eggs do not hatch when ejected from a dispenser.

**To Reproduce**
1. Build your preferred redstone ticker/clock (I used a V of blocks, two torches, a single bit of redstone dust) and dispenser setup
2. Place eggs (duck or goose) from this mod in the dispenser
3. Observe that they don't pop
4. Maybe test chicken eggs if you like

**Expected behavior**
I expected the eggs to be fired and hatch/crack/pop in the same manner as vanilla chicken eggs.

**Version**
I am running a Fabric launcher, Minecraft 1.17 pack I'm gradually putting together myself, through curseforge.

**Error Log**
There's no game-crashing error, nor anything barring the usual data fixer errors, which aren't related to this issue as far as I understand it.

**Comment by Paspartout** [comment]
Oh, I didn't notice that. Thanks for bringing this up.
Yeah the eggs should behave just like chicken eggs, this is an oversight at the moment and should be easy to fix.
I'll try to get a new release out next weekend.

**Comment by milkev** [comment]
any update on this?
